spec/mailers/notifier_spec.rb

```diff
@@ -246,7 +246,8 @@
     end
 
     it "FROM: contains the sender's name" do
-      expect(@mail["From"].to_s).to eq("\"#{@cnv.author.name} (diaspora*)\" <#{AppConfig.mail.sender_address}>")
+      pod_name = AppConfig.settings.pod_name
```

Comment:
You can write `let(:pod_name) { AppConfig.settings.pod_name }` once at the top of this file (below the `let(:person) ...` line), and then you don't need this 6 times (in every test).